File: DataAccessLayer/Model/WalletRequest.cs
using System;
using System.Collections.Generic;

namespace DataAccessLayer.Model
{
    public partial class WalletRequest
    {
        public Guid Id { get; set; }
        public Guid? UserId { get; set; }
        public double? MoneyRequest { get; set; }
        public bool? Status { get; set; }

        public virtual User User { get; set; }
    }
}

File: models/Wallet_Request.js
const mongoose = require("mongoose");
const Schema = mongoose.Schema;
require("mongoose-currency").loadType(mongoose);

const walletRequestSchema = new Schema(
  {
    user_id: {
      type: mongoose.Schema.Types.ObjectId,
      ref: "User",
      required: true,
    },
    money_request: {
      type: mongoose.Types.Currency,
      required: true,
    },
    status: {
      type: Boolean,
      required: true,
    },
  },
  {
    timestamps: true,
  }
);

var WalletRequest = mongoose.model("Wallet_Request", walletRequestSchema);

module.exports = WalletRequest;
